server/server/src/main/kotlin/org/jetbrains/bsp/bazel/server/sync/BazelProjectMapper.kt

```diff
@@ -112,6 +112,10 @@ class BazelProjectMapper(
       measure("Create android libraries") {
         calculateAndroidLibrariesMapper(targetsToImport, workspaceContext)
       }
+    val goLibrariesMapper =
```

Comment:
**suggestion (performance):** Consider performance impact of Go support

The addition of Go libraries support is a good feature, but consider adding a comment about potential performance implications for projects that don't use Go. Ensure that this addition doesn't negatively impact performance for non-Go projects.